docs/topics/mta-web-proxy-config.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * docs/cli-guide/master.adoc
+
+:_content-type: PROCEDURE
+[id="mta-web-proxy-config_{context}"]
+= Configuring HTTP and HTTPS proxy settings
+
+You can configure HTTP & HTTPS proxy settings with this management module.
```

Comment:
Write out "and" -- don't use "&" unless it is part of a UI name (or a similar situation where you are quoting an existing name).

Reply:
OK